core/src/test/java/org/kiwix/kiwixmobile/core/search/viewmodel/SearchViewModelTest.kt

```diff
@@ -226,7 +227,17 @@ internal class SearchViewModelTest {
       actionResultsInEffects(
         OnItemClick(searchListItem),
         SaveSearchToRecents(recentSearchDao, searchListItem, "id"),
-        OpenSearchItem(searchListItem)
+        OpenSearchItem(searchListItem, false)
+      )
+    }
+
+    @Test
+    fun `OnOpenInNewTabClick offers Saves and Opens in new tab`() {
+      val searchListItem = RecentSearchListItem("")
+      actionResultsInEffects(
+        OnOpenInNewTabClick(searchListItem),
+        SaveSearchToRecents(recentSearchDao, searchListItem, "id"),
+        OpenSearchItem(searchListItem, true)
       )
     }
```

Comment:
Just to note, I'm getting compile errors unrelated to what I added for some reason, so I wasn't able to test this.

My org.kiwix.sharedFunctions aren't able to import for some reason.

Reply:
yeah that is an annoying bug I have filed an issue for, if you run the tests they work but the IDE complains, you may have to manually add some imports. Seeing as how the tests pass fine I have been dealing with it because I like the freedom of being able to construct any of these models in Unit or Instrumented tests in any module, it is annoying though

Reply:
Thanks, I didn't realize the test would run anyway.